gulpfile.js

```diff
@@ -32,19 +32,6 @@ var DOCS = path.join(SRC, 'docs');
 var SASS = path.join(SRC, 'sass');
 var COMPONENTS = path.join(SRC, 'components');
 
-gulp.task('sass:docs', function () {
-    return gulp.src(path.join(DOCS, 'sass', '**', '*.scss'))
-        .pipe(gulpif(DEV_ENV, sourcemaps.init()))
-        .pipe(sass().on('error', sass.logError))
-        .pipe(autoprefixer({
-            browsers: ['last 2 versions', 'ie 8', 'ie 9'],
-            cascade: true
-        }))
-        .pipe(gulpif(DEV_ENV, sourcemaps.write()))
-        .pipe(gulp.dest(path.join(VERSIONED_DIST, 'docs', 'css')))
-        .pipe(gulpif(DEV_ENV, livereload()));
-});
-
```

Comment:
Judging from the sources it looks like a ruby gem. Haven't seen any issues with it though. I say we leave it since it only compiles docs scss files, not the styleguide itself.